src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  line-height: 1.5;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  color: $main-text-color;
+  transition: transform $transition-duration ease 0s;
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    transition: color $transition-duration ease 0s;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__code {
+    font-size: 10px;
+    margin-bottom: 16px;
+    color: $second-text-color;
+    line-height: 1.4;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 1.4;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-name {
```

Comment:
```suggestion
    margin-bottom: 16px;

  &-name {
```

Reply:
10'th point in checklist

Reply:
Still not fixed, move &-name inside &__price block